### ISSUE TITLE
[5.3.4]  Volume control natively and/or easily

### ISSUE THREAD

**Issue by Jon8RFC** [OrangeDrangon/android-messages-desktop#398]
**Is your feature request related to a problem? Please describe.**
There isn't easy access to volume control.

**Describe the solution you'd like**
An in-app volume control, which doesn't hook into the Windows "Volume Mixer".
Most Windows applications have native volume control.

**Describe alternatives you've considered**
In Windows, there is the "Volume Mixer".  However, that cannot be accessed at all times (at least in Win10).  A recent sound must be made by the application for its volume slider to be accessible.

If no native volume control...
A dirty workaround (just an idea to spawn a better one) to have the Windows "Volume Mixer" slider always accessible could be playing a periodic, short, silent audio track.  [From what I've read](https://answers.microsoft.com/en-us/windows/forum/all/application-not-appearing-in-the-volume-mixer/52c09680-4b39-490a-973d-7a880b8cce04), an "audio session" must be started for a given application for it to appear in the Volume Mixer.  It also disappears after a few minutes, presumably due to a timeout in Win10.
<details><summary>Audio length edge case</summary>
I don't know if a 0.1 second and 100% silent audio track would necessarily qualify as a proper "audio session", or for which OSes.  Maybe it will work perfectly at any length and any volume.  However, based on a bug which exists in Android 10+, frustrating edge cases do exist for "audio sessions" in operating systems 😂
</details>
⠀

**Additional context**
An easy option to change the notification sound would be a welcomed addition as well, but easy volume control is the most important thing I'm requesting.

This application is excellent.  Thank you!

**Comment by OrangeDrangon** [comment]
Are you looking for volume control to adjust notifications sounds?

**Comment by Jon8RFC** [comment]
Yes.  I'm going to be embarrassed if it's been there all along and I haven't seen it.

**Comment by OrangeDrangon** [comment]
No nothing like that exists as far as I am aware. Just a toggle between making noise and not :)

**Comment by OrangeDrangon** [comment]
> In Windows, there is the "Volume Mixer". However, that cannot be accessed at all times (at least in Win10). A recent sound must be made by the application for its volume slider to be accessible.

Is this true? From quick testing in windows the volume mixer entry seems to be present even when no sound has been made. I will leave it open for a while to see if it goes away.

**Comment by Jon8RFC** [comment]
I've had the app open for a while on a few different computer boot ups and it disappears on Win10, and won't exist until a sound has been made.  I do have "allow apps to take exclusive control of this device" unchecked, if something in the app depends on that.

I wonder if it will remain if the Volume Mixer is left open.  I've never had other applications' volume slider disappear, as far as I know.  Chrome's has never not been there for me.

**Comment by OrangeDrangon** [comment]
Yeah I just got it to go away. If I get some time I will look into doing something about this. Worst case for now you can always text yourself to get the volume slider back :)
